### PR TITLE
rename clap to cdep, add install upgrade

### PR DESCRIPTION
**Pull request by limor-gs**

### CODE REVIEW

File: scripts/cdep [hunk 1]
```diff
@@ -20,14 +20,11 @@ CORE_REPOS = [
...
     'cloudify-amqp-influxdb',
-    'docl'
-]
-DEV_REPOS = [
     'cloudify-fabric-plugin',
     'cloudify-system-tests',
     'cloudify-dev'
 ]
-REPOS = CORE_REPOS + DEV_REPOS
+REPOS = CORE_REPOS
```
Comment (mcouthon) [inline scripts/cdep]: If you only have one list, what's the point of having two names?

**Comment by mcouthon** [comment]
@limor-gs Can we close this?

**Comment by mcouthon** [comment]
@limor-gs Can I close? I think this is going to stay clap :)

**Comment by mcouthon** [comment]
@adigrabow Can you please close this? :)

**Comment by tehasdf** [comment]
@mcouthon can I? 😛

**Pull request closed without merging**